rust-connectors/sinks/sql/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "sql-sink"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
is this published? if not, no need to bump

Reply:
Yes, https://hub.docker.com/r/infinyon/fluvio-connect-sql-sink/tags

Reply:
got it.